src/DBus.vala

```diff
@@ -110,6 +110,19 @@ public class Notifications.Server : Object {
         } else {
             var notification = new Notifications.Notification (app_name, app_icon, summary, body, actions, hints);
 
+            // Silence "Automatic suspend. Suspending soon because of inactivity." notifications
+            // These values and hints are taken from gnome-settings-daemon source code
+            // See https://gitlab.gnome.org/GNOME/gnome-settings-daemon/-/blob/master/plugins/power/gsd-power-manager.c#L356
+            // We must check for app_icon == "" to not block low power notifications
+            if (hints.contains ("desktop-entry") && ((string) hints.get ("desktop-entry")) == "gnome-power-panel" &&
+                hints.contains ("x-gnome-privacy-scope") && ((string) hints.get ("x-gnome-privacy-scope")) == "system" &&
```

Comment:
i don't think we should check the privacy scope, if the user change the sleep action to be logout, this will let pass the "Automatic logout" notification, since that use the user scope.

in fact, all we need is the "desktop-entry" and app_icon check. that should be enough to not catch others notifications that we don't want to block.

Reply:
Fixed.